CONTRIBUTING.md

```diff
@@ -0,0 +1,117 @@
+# Contributing
+
+When contributing to Figura, please first discuss and/or interact with our community via our [Discord](https://discord.gg/figuramc). 
+
+Please note we have a code of conduct, please follow it in all your interactions with the project.
+
+## Pull Request Process
+note - do not merge upstream into your pr branch if a clean rebase is possible
+
+1. Ensure the feature is In demand via the `📧丨mod-suggestions` channel in our [Discord](https://discord.gg/figuramc) or has the [Enhancment]([https://github.com/FiguraMC/Figura/labels/enhancement](https://github.com/FiguraMC/Figura/issues?q=is%3Aissue+is%3Aopen+label%3Aenhancement+)) tag in the Issues tab 
+   
+    OR:
+
+   Ensure what you are doing is actually a bug via the `👾丨bug-reports` channel in our [Discord](https://discord.gg/figuramc) or has the [Bug]([https://github.com/FiguraMC/Figura/issues](https://github.com/FiguraMC/Figura/issues?q=is%3Aissue+is%3Aopen+label%3Abug+)) tag in the Issues tab 
```

Comment:
Same here, although confirming that something *is* a bug is more important as there are intentional limitations in multiple places.